packages/nxengine/build.sh

```diff
@@ -0,0 +1,39 @@
+TERMUX_PKG_NAME=NXEngine
```

Comment:
You can't change this variable. Also package name must be lower case.

Reply:
I see,